elsa-analyser.el

```diff
@@ -454,7 +454,7 @@ make it explicit and precise."
     (unless var-type
       (if value
           (progn
-            (message "analyzer: updating variable %s, old type %s" var-name (elsa-type-describe var-type))
+            ;; (message "analyzer: updating variable %s, old type %s" var-name (elsa-type-describe var-type))
```

Comment:
Probably should just remove this line, no?

Reply:
Yea heh, it got into master by accident probably.  I should just delete it there and rebase.